tests/invert_cases/additionalProperties/any_true.json

```diff
@@ -0,0 +1,8 @@
+{
+  "schema": {
+    "additionalProperties": true
+  },
+  "inverted": {
```

Comment:
I think this should be simply `false`.

Reply:
Maybe this is meant to be inverting `{"anyAdditionalProperties": true}`.

Reply:
Yeah that is incorrect. Let me see why this test isn't failing.